core/src/commonMain/kotlin/com/powersync/db/crud/CrudEntry.kt

```diff
@@ -60,7 +61,12 @@ public data class CrudEntry(
                 id = data["id"]!!.jsonPrimitive.content,
                 clientId = row.id.toInt(),
                 op = UpdateType.fromJsonChecked(data["op"]!!.jsonPrimitive.content),
-                opData = data["data"]?.jsonObject?.mapValues { it.value.jsonPrimitive.content },
+                opData = data["data"]?.jsonObject?.mapValues { (_, value) ->
+                    when {
+                        value.jsonPrimitive.contentOrNull != null -> value.jsonPrimitive.content
+                        else -> null
+                    }
```

Comment:
Can this be simplified to just `value.jsonPrimitive.contentOrNull`?

Reply:
yes! thanks